lib/msal-browser/src/config/Configuration.ts

```diff
@@ -204,7 +215,7 @@ export type Configuration = {
 };
 
 export type BrowserConfiguration = {
-    auth: Required<BrowserAuthOptions>;
+    auth: Required<InternalAuthOptions>;
```

Comment:
Your type definition above already makes everything required
```suggestion
    auth: InternalAuthOptions;
```